src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.scss

```diff
@@ -61,3 +61,17 @@
     padding: 10px 0 20px;
   }
 }
+
+.findAGene {
+  color: $blue;
+  cursor: pointer;
+  font-size: 13px;
+  padding: 40px 0 20px 10px;
```

Comment:
The vertical distance in the XD looks smaller than the 40px padding produces. Could you check with Andrea; maybe she will change her mind?

![image](https://user-images.githubusercontent.com/6834224/175879477-0a875ef1-8f96-4c42-8b77-d9e252ef504c.png)

Reply:
change to 30px